app/src/main/java/com/example/iou/bill/activities/DividingItemsActivity.java

```diff
@@ -0,0 +1,76 @@
+package com.example.iou.bill.activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.LinearLayout;
+
+import com.example.iou.R;
+import com.example.iou.bill.adapters.BillItemAdapter;
+import com.example.iou.bill.models.BillItem;
+import com.example.iou.bill.models.SplitBill;
+
+import org.parceler.Parcels;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DividingItemsActivity extends AppCompatActivity {
+
+    private SplitBill splitBill;
+    private List<BillItem> items;
+    private BillItemAdapter adapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_dividing_items);
+
+        final Button btnDivideNext = findViewById(R.id.btnDivideNext);
+        final RecyclerView rvItems = findViewById(R.id.rvItems);
+
+        // Unwrap the information from the Split Information Activity
+        splitBill = Parcels.unwrap(getIntent().getParcelableExtra("Split Bill Information"));
```

Comment:
we can probably extract "Split Bill Information" out into a separate variable as I imagine we're using the same key to put the parcel in.

Ideally we'd want a separate class that tracks shared keys :)

It's also the convention that we use snake case for these things (e.g. "SPLIT_BILL_INFORMATION")

`private final string SPLIT_BILL_INFORMATION_KEY = "SPLIT_BILL_INFORMATION`